downstream/modules/platform/con-controller-task-manager-settings.adoc

```diff
@@ -0,0 +1,21 @@
+[id="con-controller-task-manager-settings"]
+
+= Task Manager (Job Scheduling) Settings
+
+The task manager is a periodic task that collects tasks to be scheduled and determines what instances have capacity and are eligible to run them. 
+Its job is to find and assign the control and execution instances, update the job's status to waiting, and send the message to the control node through `pg_notify` for the dispatcher to pick up the task and start running it.
+
+//Avoid talking about what used to happen.
+//As mentioned in xref:ref-controller-performance-improvements[Performance improvements[], a number of optimizations and refactors of this process were implemented in version 4.3. One such refactor was to fix a defect that when the task manager did reach its timeout, it was terminated in such a way that it did not make any progress. Multiple changes were implemented to fix this, so that as the task manager approaches its timeout, it makes an effort to exit and commit any progress made on that run. These issues generally arise when there are thousands of pending jobs, so may not be applicable to your use case.
```

Comment:
I think this paragraph is hidden but it looks like there is and extra bracket after "Performance improvements[]"